lms/services/product.py

```diff
@@ -0,0 +1,24 @@
+from lms.models import Product
+
+
+def _get_product(request):
+    if request.content_type == "application/json":
+        # We are in an API request, where we are forwarding the product type ourselves
+        product_name = request.json["lms"]["product"]
+
+    else:
+        # In an LTI launch we'll use the parameters available to guess
+        product_name = request.lti_params.get("tool_consumer_info_product_family_code")
```

Comment:
Using the new `request.lti_params` here. 

We could build LTIParams here but I feel we'd be duplicating a simple but potentially easy to get wrong code.